src/quo/components/wallet/transaction_summary/schema.cljs

```diff
@@ -0,0 +1,23 @@
+(ns quo.components.wallet.transaction-summary.schema
+  (:require [quo.components.tags.context-tag.schema :as ?context-tag]))
+
+(def ?schema
+  [:=>
+   [:catn
+    [:props
+     [:map
+      [:theme :schema.common/theme]
+      [:transaction {:optional true} [:maybe [:enum :send :swap :bridge]]]
+      [:first-tag {:optional true} [:maybe ?context-tag/?schema]]
```

Comment:
Ns imported with `?` feels really off. Maybe others can also comment on this ? 
I'd rather have it as `context-tag-schema` or `ct-schema`.

The reason is that varialbles with a `?` prefix are conventionally Malli Schemas. But `?context-tag` is not a schema. It's a namespace.

Reply:
we usually import it by aliasing as `component-schema`

Reply:
Done

Reply:
@clauxx I don't agree with `component-schema` since `component` here might be confusing and refers to the current component. We are referring to another component